### PR TITLE
chore: :pencil2: Small edits and fixes to (mostly) redundant stuff

### PR DESCRIPTION
**Pull request by lwjohnst86**
This is more a housekeeping Pull Request.

### CODE REVIEW

**Review by K-Beicher — APPROVED** [review]
Looks good.

File: tools/pr/reformat-markdown.sh
Comment (signekb) [inline tools/pr/reformat-markdown.sh:1]: How do we reformat markdown+quarto files now with e.g., column length?

**Review by signekb — APPROVED** [review]
👍

File: tools/pr/reformat-markdown.sh
Comment (lwjohnst86) [reply tools/pr/reformat-markdown.sh:1]: haha we didn't really do that anyway, even with this file. I don't know yet how to do it, I haven't found an easy/convenient way :scream:

Comment (signekb) [reply tools/pr/reformat-markdown.sh:1]: @lwjohnst86 Yeah, I couldn't find one either... Very annoying. Should we just build an extension for it, haha

Comment (lwjohnst86) [reply tools/pr/reformat-markdown.sh:1]: I think we will have to HAHAHA

**Pull request merged**